RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -759,8 +766,12 @@ private static void performCLIExit(RoddyStartupModes option, int exitCode) {
 
     private static int waitForJobs() {
         try {
-            Thread.sleep(15000); //Sleep at least 15 seconds to let any job scheduler handle things...
-            return jobManager.waitForJobsToFinish();
+            if (jobManager.isDaemonAlive()) {
+                logger.always("Roddy will wait for jobs now.");
+                Thread.sleep(5000); //Sleep at least 5 seconds to let any job scheduler handle things...
+                return jobManager.waitForJobsToFinish();
+            }
+            return 249;
```

Comment:
Can we use  a constant describing the meaning of 249 and 250?
Please add this to the RtD documentation.

Reply:
Sure, I'll implement them.

Reply:
Yes, overlooked it.